gtsfm/averaging/translation/averaging_1dsfm.py

```diff
@@ -64,7 +67,12 @@ def run(
                 or ill-constrained system).
             A GtsfmMetricsGroup of 1DSfM metrics.
         """
-        noise_model = gtsam.noiseModel.Isotropic.Sigma(NOISE_MODEL_DIMENSION, NOISE_MODEL_SIGMA)
+        isotropic_noise_model = gtsam.noiseModel.Isotropic.Sigma(NOISE_MODEL_DIMENSION, NOISE_MODEL_SIGMA)
```

Comment:
just rename the first variable as noise_model, and overwrite it if the flag is true?